source/features/parse-backticks.tsx

```diff
@@ -4,47 +4,116 @@ import * as pageDetect from 'github-url-detection';
-	for (const title of select.all([
+function parse(selectors: string[]): void {
+	for (const element of select.all(selectors.map(selector => selector + ':not(.rgh-backticks-already-parsed)'))) {
+		element.classList.add('rgh-backticks-already-parsed');
+		parseBackticks(element);
+	}
+}
+
+function initRepo(): void {
+	parse([
 		'.commit-title', // `isCommit`
 		'.commit-desc', // `isCommit`, `isCommitList`, `isRepoTree`
 		'.commit-message', // Pushed commits in `isPRConversation`, `isCompare`, `isReleasesOrTags`
 		'.message', // `isCommitList`, `isRepoTree`, `isBlame`
-		'.Box--condensed .link-gray[href*="/commit/"]', // `isSingleFile`
+		'.repository-content .js-details-container .link-gray[href*="/commit/"]', // `isSingleFile`
+		'.repository-content .js-details-container pre', // `isSingleFile`
 		'[aria-label="Issues"][role="group"] .js-navigation-open', // `isConversationList`
 		'[id^=ref-issue-]', // Issue references in `isIssue`, `isPRConversation`
 		'[id^=ref-pullrequest-]', // PR references in `isIssue`, `isPRConversation`
 		'.TimelineItem-body > del, .TimelineItem-body > ins', // Title edits in `isIssue`, `isPRConversation`
-		'.js-pinned-issue-list-item > span', // Pinned Issues
+		'.js-pinned-issue-list-item > .d-block', // Pinned Issues
 		'.pulse-section li', // `isPulse`
-		'.issues_labeled .text-gray-dark > a', // `isDashboard` "help wanted" event titles
-		'.js-recent-activity-container .text-bold', // `isDashboard`"Recent activity" titles
-		'.commits blockquote', // Newsfeed commits
 		'.release-header', // `isReleasesOrTags` Headers
-		'.Box-row.js-navigation-item a.link-gray-dark', // `isGlobalConversationList`
-		'[data-channel^="check_suites"] a', // `isActions`
+		'[id^="check_suite"] a.link-gray-dark', // `isActions`
 		'.repository-content .pr-toolbar h2', // `isActions` run
-		'#wiki-wrapper h1' // `isWiki`
-	].map(selector => selector + ':not(.rgh-backticks-already-parsed)'))) {
-		title.classList.add('rgh-backticks-already-parsed');
-		parseBackticks(title);
-	}
+		'.Details[data-issue-and-pr-hovercards-enabled] .Details-content--hidden a.link-gray-dark', // `isRepoRoot`
+		'.Details[data-issue-and-pr-hovercards-enabled] .Details-content--hidden pre', // `isRepoRoot`
+		'.Details[data-issue-and-pr-hovercards-enabled] .d-none a.link-gray-dark', // `isRepoRoot`
+		'.existing-pull-contents .list-group-item-link', // `isCompare` with existing PR
+		'[aria-label="Link issues"] a', // "Linked issues" in `isIssue`, `isPRConversation`
+		'.BorderGrid--spacious .f4.mt-3', // `isRepoHome` repository description
+		'.js-wiki-sidebar-toggle-display a', // `isWiki`
+		'.gh-header-title' // `isWiki`
+	]);
+}
+
+function initDashboard(): void {
+	parse([
+		'.js-recent-activity-container .text-bold', // `isDashboard`"Recent activity" titles
+		'.commits blockquote' // Newsfeed commits
+	]);
+}
+
+function initNotifications(): void {
+	parse([
+		'.notifications-list-item p.text-normal' // `isNotifications`
```

Comment:
```suggestion
		'.notifications-list-item p.text-normal'
```
Don't think there is a need for a comment